src/selection_stream/event_stream.ts

```diff
@@ -85,6 +85,14 @@ export class EventStream<Event> {
     this.mainSubscription = this.defaultSubscription;
   }
 
+  /**
+   * Release whichever subscription in charge and get back to the default subscription
+   */
+  getBackToDefault() {
+    this.mainSubscription?.callbacks.release?.();
```

Comment:
I wonder if we should *not* call the release when the mainsubscription is the same a the default one.